dashboard/admin.py

```diff
@@ -115,8 +116,14 @@ def save_model(self, request, obj, form, change):
         return super(CourseAdmin, self).save_model(request, obj, form, change)
 
 class LogResource(resources.ModelResource):
+
+    # This field is used to get the username from the user model
```

Comment:
We have a model named literally meaning `user` and other by `auth_user`. For clarity purposes we can update the comment saying `auth_user`

Reply:
```suggestion
    # This field is used to get the username from the auth_user model
```